openeo_plugin/gui/browser/OpenEOConnectionItem.py

```diff
@@ -53,11 +54,24 @@ def createChildren(self):
     def remove(self):
         self.parent().removeConnection(self)
 
+    def authenticate(self):
+        self.dlg = LoginDialog(
+            connection=self.model,
+            iface=self.plugin.iface
+            )
+        
+        result = self.dlg.exec()
+
+        if result:
+            return
+
     def getConnection(self):
         return self.connection
     
     def actions(self, parent):
+        action_authenticate = QAction(QIcon(), "Authenticate Connection", parent)
```

Comment:
I'm wondering whether this would be a bit more inuitive for some folks. Not sure how commonly known the word Authentication is?!

```suggestion
        action_authenticate = QAction(QIcon(), "Authentication (Login)", parent)
```